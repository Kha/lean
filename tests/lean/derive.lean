@[derive [decidable_eq, inhabited, has_sizeof]]
inductive foo
| bar : ℕ → foo
| baz : foo

#check foo.decidable_eq
#check foo.inhabited
#check foo.has_sizeof
