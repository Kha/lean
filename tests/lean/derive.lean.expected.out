foo.decidable_eq : decidable_eq foo
foo.inhabited : inhabited foo
foo.has_sizeof : has_sizeof foo
